ui/app/AppLayouts/Communities/stores/CommunitiesStore.qml

```diff
@@ -100,6 +100,10 @@ QtObject {
         return root.communitiesModuleInst.getCommunityPublicKeyFromPrivateKey(privateKey);
     }
 
+    function scanCommunitiesMembersForName(displayName) {
```

Comment:
Could use a better name... it doesn't really tell the QML user that this returns a boolean and moreover what's the meaning of it

Reply:
`isDisplayNameOccursAmongCommunitiesMembers`?

Reply:
OK, then what about `isDisplayNameDupeOfCommunityMember`?